go

```diff
@@ -224,7 +239,7 @@ end
 
 command_group :cleanup, 'Image and container cleanup commands'
 
-def_command :rm_containers, 'Remove stopped (non-data) containers' do |args|
+def_command :rm_containers, 'Remove stopped non-data containers' do |args = []|
```

Comment:
Uhh, I don't think you can have defaults for block parameters in Ruby. Just tried this in IRB:

```ruby
> [1,2,nil,3].each do |el = 'foo'|
*  puts el.inspect
* end
1
2
nil
3
```